chapters/introduction.tex

```diff
@@ -163,59 +160,76 @@ \section{Writing reproducible code in a collaborative environment}
 it should not require arcane reverse-engineering
 to figure out what a code chunk is trying to do.
 \textbf{Style}, finally, is the way that the non-functional elements of your code convey its purpose.
-Elements like spacing, indentation, and naming (or lack thereof) can make your code much more 
+Elements like spacing, indentation, and naming (or lack thereof) can make your code much more
 (or much less) accessible to someone who is reading it for the first time and needs to understand it quickly and correctly.
 
```

Comment:
Add here (also addressing #355):

As you gain experience in coding and get more confident with the way you implement these suggestions, you will feel more empowered to apply critical thinking to the way you handle data. For example, you will be able to predict which section of your script are more likely to create errors. This may happen intuitively, but you will improve much faster as a coder if you do it purposefully. Ask yourself, as you write code and explore results: Do I believe this number? What can go wrong in my code? How will missing values be treated in this command? What would happen if more observations would be added to the data set? Can my code be made more efficient or easier to understand?

Reply:
7debf2059bb9053a0beb6aca3038451cf23a4aee